packages/office-ui-fabric-react/src/components/Coachmark/examples/Coachmark.Basic.Example.tsx

```diff
@@ -34,123 +29,100 @@ export interface ICoachmarkBasicExampleStyles {
-              { key: 'H', text: 'Bottom Auto Edge', data: DirectionalHint.bottomAutoEdge },
-              { key: 'I', text: 'Left Top Edge', data: DirectionalHint.leftTopEdge },
-              { key: 'J', text: 'Left Center', data: DirectionalHint.leftCenter },
-              { key: 'K', text: 'Left Bottom Edge', data: DirectionalHint.leftBottomEdge },
-              { key: 'L', text: 'Right Top Edge', data: DirectionalHint.rightTopEdge },
-              { key: 'M', text: 'Right Center', data: DirectionalHint.rightCenter },
-              { key: 'N', text: 'Right Bottom Edge', data: DirectionalHint.rightBottomEdge },
-            ]}
-            onChange={this._onDropdownChange}
-          />
-        </div>
-
-        <div className={classNames.buttonContainer} ref={this._targetButton}>
-          <DefaultButton
-            onClick={this._onShowMenuClicked}
-            text={isCoachmarkVisible ? 'Hide Coachmark' : 'Show Coachmark'}
-          />
-        </div>
-        {isCoachmarkVisible && (
-          <Coachmark
-            target={this._targetButton.current}
-            positioningContainerProps={{
-              directionalHint: this.state.coachmarkPosition,
-              doNotLayer: false,
-            }}
-            ariaAlertText="A Coachmark has appeared"
-            ariaDescribedBy={'coachmark-desc1'}
-            ariaLabelledBy={'coachmark-label1'}
-            ariaDescribedByText={'Press enter or alt + C to open the Coachmark notification'}
-            ariaLabelledByText={'Coachmark notification'}
-          >
-            <TeachingBubbleContent
-              headline="Example Title"
-              hasCloseButton={true}
-              closeButtonAriaLabel="Close"
-              primaryButtonProps={buttonProps}
-              secondaryButtonProps={buttonProps2}
-              onDismiss={this._onDismiss}
-              ariaDescribedBy={'example-description1'}
-              ariaLabelledBy={'example-label1'}
-            >
-              Welcome to the land of Coachmarks!
-            </TeachingBubbleContent>
-          </Coachmark>
-        )}
-      </div>
-    );
-  }
-
-  private _onDismiss = (): void => {
-    this.setState({
-      isCoachmarkVisible: false,
-    });
-  };
-
-  private _onDropdownChange = (event: React.FormEvent<HTMLDivElement>, option: IDropdownOption): void => {
-    this.setState({
-      coachmarkPosition: option.data,
-      dropdownSelectedOptionKey: option.key,
-    });
+const getClassNames = classNamesFunction<{}, ICoachmarkBasicExampleStyles>();
+const classNames = getClassNames(() => {
+  return {
+    dropdownContainer: {
+      maxWidth: '400px',
+    },
+    buttonContainer: {
+      marginTop: '30px',
+      display: 'inline-block',
+    },
   };
+}, {});
+
+const buttonProps: IButtonProps = {
+  text: 'Try it',
+};
+
+const buttonProps2: IButtonProps = {
+  text: 'Try it again',
+};
+
+const IDropdownOption = [
```

Comment:
Probably not the name you meant to use :)
```suggestion
const dropdownOptions: IDropdownOption[] = [
```